src/main/java/com/openlattice/shuttle/AssociationDefinition.java

```diff
@@ -66,6 +66,7 @@ public AssociationDefinition(
                     Map<FullQualifiedName, PropertyDefinition> propertyDefinitions,
             @JsonProperty( Constants.CONDITIONS ) Optional<Conditions> condition,
             @JsonProperty( SerializationConstants.NAME ) String alias,
+            @JsonProperty( Constants.GENERATOR ) Optional<SerializableFunction<Map<String, Object>, String>> generator,
```

Comment:
does this work from yaml integrations?

Reply:
That's the goal and it works with the test yaml.  
The transformValueMapper is `SerializableFunction<Map<String, Object>, String>`, so generator can be any combination of transforms.